### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by PencilNavigator**
### Questions
- [x] Did you test your changes or double check that they work?
- [x] Did you read and follow the [Atlas Contribution Guidelines](https://docs.atlasos.net/contributions/)?
- [ ] Did you commit to the [`dev`](https://github.com/Atlas-OS/Atlas/tree/dev) branch and not [`main`](https://github.com/Atlas-OS/Atlas)?

**Note:** You should commit directly to `main` for translations of the `README.md`.

### Describe your pull request
Originally https://github.com/Atlas-OS/Atlas/pull/815

Plus these following PR's
https://github.com/Atlas-OS/Atlas/pull/820
https://github.com/Atlas-OS/Atlas/pull/817
https://github.com/Atlas-OS/Atlas/pull/816

Only Format is checked, Please check the Translated Content + Possible errors.

### CODE REVIEW

**Review by Xyueta — APPROVED** [review]
quite good

**Review by he3als — APPROVED** [review]

**Review by Xyueta — APPROVED** [review]

File: translations/README_ar_SA.md [hunk 1]
```diff
@@ -17,12 +30,11 @@
 
 ## 🤔 **مَا هُو أطلس؟**
 
-أطلس هو نسخة مُعدلة من ويندوز ١٠ لمعالجة أغلب العُيوب والمشاكل التي قد تؤثّر سلبًا على أداء جهازك عند تشغيل الألعاب. نحن أيضًا خيار رائع لتقليل وقت استجابة النظام والشّبكة (latency) وتحسين زمن الإدخال (input lag) مع الحفاظ على خُصوصيّة نظامك والإبقاء على تركيزنا الأساسي ألا وهو الأدَاء. <br /> يمكنك معرفة المزيد عن أطلس من خلال [موقعنا الرسمي](https://atlasos.net).
-
-## 📚 **جدول المُحتوى:**
```
Comment (pewpewded) [inline translations/README_ar_SA.md:22]: i believe you deleted this by mistake; it's the table of contents heading.

File: translations/README_ar_SA.md [hunk 2]
```diff
@@ -17,12 +30,11 @@
...
+أطلس هو نسخة مُعدلة من ويندوز لمعالجة أغلب العُيوب والمشاكل التي قد تؤثّر سلبًا على أداء جهازك عند تشغيل الألعاب. نحن أيضًا خيار رائع لتقليل وقت استجابة النظام والشّبكة (latency) وتحسين زمن الإدخال (input lag) مع الحفاظ على خُصوصيّة نظامك والإبقاء على تركيزنا الأساسي ألا وهو الأدَاء. يمكنك معرفة 
+المزيد عن أطلس من خلال [موقعنا الرسمي](https://atlasos.net).
 
 <ul>
-<li>البداية:
+<li>البداية📚
```
Comment (pewpewded) [inline translations/README_ar_SA.md:37]: revert this, to match to the style of the rest list items.

**Comment by pewpewded** [comment]
i'll work on the mentioned comments

https://github.com/Atlas-OS/Atlas/pull/820#issuecomment-1575514620

**Pull request merged**